lib/ansible/plugins/inventory/azure_rm.py

```diff
@@ -446,6 +446,16 @@ def send_request(self, url, api_version):
         return json.loads(content)
 
 
+def azure_id_to_dict(id):
```

Comment:
no generic function available? or not sufficient?

Reply:
i mean:     from msrestazure.tools import parse_resource_id, resource_id, is_valid_resource_id